packages/docs/node-operations/key-management.md

```diff
@@ -1,83 +0,0 @@
-| Attestation signer key | This key is used to sign attestations in Celo's lightweight identity protocol.                                                                                                                                                                                        |
-
-{% hint style="info" %}
-Each signer key must be unique and may not be reused. Once a signer key is authorized, the only way to deauthorize that key is to authorize a new key that has never been used before.
-{% endhint %}
-
-# Key Rotation
-
-Loss of an authorized key is not catastrophic, as the Account key can just authorize another key in its place. It is strongly recommended to regularly rotate keys to limit the impact of an undiscovered compromise.
-
-### Key Rotation for Consensus
-
-Key rotation for Consensus is a bit trickier. As a validator that is currently elected and actively signing consensus messages, you can rotate the Validator key as follows:
-
-- Bring up a new validator node and create a new Validator signer key on it.
-- Authorize the new Validator signer key with the Account key to overwrite the old Validator signer key.
-
-```bash
-celocli account:authorize --from <ACCOUNT_KEY_ADDRESS> --role validator --signer <NEW_VALIDATOR_ADDRESS> --signature <PROOF_OF_NEW_VALIDATOR_KEY_POSSESSION>
-```
-
-{% hint style="info" %}
-Your ECDSA public key will be updated automatically from the proof-of-possession.
-{% endhint %}
-
-- Update your BLS key used for consensus. Rotating both the ECDSA and BLS keys is recommended.
-
-```bash
-celocli validator:update-bls-public-key --from <NEW_VALIDTOR_KEY_ADDRESS> --blsKey <NEW_BLS_KEY> --blsPop <PROOF_OF_NEW_BLS_KEY_POSSESSION>
-```
-
-- Your old node will continue to sign consensus messages for the current epoch, but upon the next epoch will find itself no longer authorized.
-- Your new node will initially find itself unauthorized, but will be authorized to sign as the next epoch begins.
-
-{% hint style="warning" %}
-The newly authorized keys will only take effect in the next epoch, so the node operating with the old key must remain running until the end of the current epoch to avoid downtime.
-{% endhint %}
-
-Please see the [Running a Validator](https://docs.celo.org/getting-started/baklava-testnet/running-a-validator) section for details on ECDSA and BLS key generation, authorization, and proof of possession.
-
-# Key Security
-
-It is evident that the Account key is the most sensitive key. As it should be used quite infrequently, it is highly recommended for important Account keys (i.e. Validators or accounts with high balances) to remain as secure as possible. At the minimum, we recommend them to be offline, ideally in cold storage or on a hardware wallet.
-
-<!-- NOTE: Unclear if this is the right place for the following table, or if it should exist based on how difficult it will be to accuratly maintain -->
-
-# Key Usage in Protocol
```

Comment:
Did you mean to not copy these sections over?

Reply:
Yes, I decided that they were too low level to be there and I couldn't really think of a good place to put them. These kinds of tables are nice, but bit rot quickly.